fs/zbd_stat.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment:
Is this file expected to be added in this PR?

Reply:
Sorry, this file will be used for another PR.